spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -27,8 +27,10 @@
 
 import requirements
 
+from dependencies import dependencies
```

Comment:
One quick comment: I noticed our RTD documentation build is failing for this PR, and the error seems to be:

WARNING: Unexpected return code 1 from command Command(command=('sct_check_dependencies', '-h'), shell=False, hide_standard_error=False, working_directory='/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/checkouts/3646/documentation/source') (output='Traceback (most recent call last):\n  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/3646/lib/python3.7/site-packages/spinalcordtoolbox/scripts/sct_check_dependencies.py", line 30, in <module>\n    from dependencies import dependencies\nModuleNotFoundError: No module named \'dependencies\'')

Highlighting the line here.

Reply:
I updated the PR but the CI has been stuck for 18h, is that normal?

Reply:
Very sorry about this! 

![Screenshot from 2022-01-14 12-24-29](https://user-images.githubusercontent.com/16181459/149558354-f800c7a1-73c7-42d4-a9ab-750aa2512311.png)

This is something that GitHub enforces, and there's no settings in our admin panel to stop it from happening. As soon as your first SCT PR is merged, CI runs will start immediately. :)